src/api-reference/common/list-item/v4.list-item-bulk.markdown

```diff
@@ -0,0 +1,494 @@
+---
+title: List Item Bulk v4
+layout: reference
+---
+
+The SAP Concur List Item Bulk API provides an automated solution to clients who would like to add, update, and delete list items. Use of the API is subject to some limitations on the volume of list data. You may need to manage the initial load of large volumes of data via a file import due to capacity limitations. This is also true if ongoing maintenance of list values involves a large volume.
+
+> **Limitations**: This API is only available to partners who have been granted access by SAP Concur. Access to this documentation does not provide access to the API.
+
+## Contents
+
+* [Process Flow](#process-flow)
+* [Products and Editions](#products-editions)
+* [Scope Usage](#scope-usage)
+* [Dependencies](#dependencies)
+* [Access Token Usage](#access-token-usage)
+* [Create List Items](#post-list-item-bulk)
+* [Update List Items](#patch-list-item-bulk)
+* [Schema](#schema)
+  * [Bulk List Item Create Request](#schema-bulk-list-item-create-request)
+  * [Bulk List Item Create Request Part](#schema-bulk-list-item-create-request-part)
+  * [Bulk List Item Update Request](#schema-bulk-list-item-update-request)
+  * [Bulk List Item Update Request Part](#schema-bulk-list-update-request-part)
+  * [Bulk List Item Response](#schema-bulk-list-item-response)
+  * [Bulk List Error](#schema-bulk-list-error)
+  * [Bulk List Item Request](#schema-bulk-list-request)
+  * [Error Message](#schema-error-message)
+  * [Validation Error](#schema-validation-error)
+  * [Message](#schema-message)
+
+## Prior Versions
+
+* List Items v1 documentation is available [here](https://developer.concur.com/api-reference/common/list-item/v1.list-item.html).
+
+## <a name="process-flow"></a>Process Flow
+
+![A process flow diagram of the List Item Bulk API](./v4-list-item-bulk-process-flow.png)
+
+## <a name="products-editions"></a>Products and Editions
+
+* Concur Expense Professional Edition
+* Concur Expense Standard Edition
+* Concur Invoice Professional Edition
+* Concur Invoice Standard Edition
+* Concur Request Professional Edition
+* Concur Request Standard Edition
+
+## <a name="scope-usage"></a>Scope Usage
+
+|Name|Description|Endpoint|
+|---|---|---|
+|`spend.listitem.write`|Read and write access to spend list items.|GET, POST, PUT, PATCH|
+|`spend.listitem.delete`|Delete capabilities for spend list items.|DELETE|
```

Comment:
`delete` scope isn't actually required for these two api's, only `write` access is required.